tests/baselines/reference/exportLocalSpellingSuggestion.errors.txt

```diff
@@ -0,0 +1,14 @@
+tests/cases/compiler/b.ts(1,10): error TS2460: Module '"./a"' declares 'assertNever' locally, but it is exported as 'assertNevar'.
+
+
+==== tests/cases/compiler/a.ts (0 errors) ====
+    declare function assertNever(): void;
+    declare function foo(): void;
+    export { foo };
+    
+==== tests/cases/compiler/b.ts (1 errors) ====
+    import { assertNevar } from "./a";
+             ~~~~~~~~~~~
+!!! error TS2460: Module '"./a"' declares 'assertNever' locally, but it is exported as 'assertNevar'.

```

Comment:
I don't think this is the correct error - `assertNever` isn't actually exported at all. This error message was meant more for the following case

```ts
// @filename: ./fileA.ts
function foo() {
}

export { foo as bar };

// @filename ./fileB.ts
import { foo } from "./fileA";
foo();
```

Reply:
@DanielRosenwasser Thanks for your feedback. I used the example from the issue description. 

```ts
// @filename a.ts

declare function foo(): any
declare function bar(): any;
export { bar };

// @filename b.ts
import { foo, bar } from "./a";

// Module '"./a"' declares 'foo' locally, but it is not exported.
// Module '"./a"' declares 'bar' locally', but it is exported as 'baz'.
```

Could you provide a couple of examples for the two error messages?

Reply:
I think the example you gave is perfect, but the error message this thread is pointing to isn't correct, is it? `assertNever` isn't actually exported at all, especially not with the name `assertNevar`.